dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops

```diff
@@ -185,3 +185,11 @@ trusted.header.origin.invalid.value.useraction=Ensure that configured IP address
 trusted.sensitive.header.origin.invalid.value=CWWKT0039W: The [{0}] configured trustedSensitiveHeaderOrigin value is invalid and cannot be used.
 trusted.sensitive.header.origin.invalid.value.explanation=The specified value must be either a full IP address or correctly formatted hostname.
 trusted.sensitive.header.origin.invalid.value.useraction=Ensure that configured IP addresses contain a value for each segment, and that hostnames do not contain any invalid characters.
+
+config.hsts.invalid.max.age.value=CWWKT0040W: The configured max-age value [{0}] is invalid and cannot be used. The max-age value must be a number equal to or larger than 0.
```

Comment:
The configured max-age value [{0}] is invalid and cannot be used. The max-age value must be a number equal to or larger than 0.
-->
The [{0}] configured max-age value  is invalid and cannot be used. The max-age value must be a number that is greater than or equal to 0.

Reply:
I've simplified this to be a non-negative integer (as a general indication of what is considered valid for the max-age value). You'll notice the rest of the messages updated on a similar fashion. Are you ok with this alternate way of specifying it? It makes it easier to understand that for the configuration to be valid:
1. max-age must be defined
2. max-age must have a non-negative integer value ( 0, 1, 2 ... for instance)

a valid configuration for instance would be something like:

 <webContainer addstricttransportsecurityheader="max-age=12345; "   />
 
 OR
 
 
 <webContainer addstricttransportsecurityheader="max-age=0; includeSubDomains"   />